tests/runtest.proj

```diff
@@ -288,10 +288,14 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
                     {
                       sErrorText = "Unable to read error file: " + errorFile%3B
                     }
+                    System.IO.StreamReader outputReader = new System.IO.StreamReader(outputFile)%3B
+                    String outputText = outputReader.ReadToEnd()%3B
+                    outputReader.Close()%3B
 
                     string msg = infraEx != null ? "Test Infrastructure Failure: " + infraEx.Message
                                                  : sErrorText + "\n\n" +
-                                                   "Raw output:      " + outputFile + "\n" +
+                                                   "Raw output file:      " + outputFile + "\n" +
```

Comment:
Please also print the actual test exit code in the error message (i.e. the value of the ret variable).